examples/nextjs/.babelrc

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
Is there a reason why all the Nextjs example has been deleted?

Reply:
If I understand it correctly, it uses https://github.com/timkendrick/monaco-editor, which was originally designed to make builds easier. Since Microsoft now provides https://github.com/Microsoft/monaco-editor-webpack-plugin, I figured we could go fully for that and focus on supporting one thing but doing it well. That's also why I removed support for AMD. 

If you think that's not a good idea, I'm happy to change this PR.

Reply:
That's ok, I was just curious

Reply:
I should have clarified this before sending you the PR.